autopush/endpoint.py

```diff
@@ -744,8 +744,9 @@ def post(self, uaid=""):
         self.start_time = time.time()
         self.add_header("Content-Type", "application/json")
         params = self._load_params()
+        # If the client didn't provide a CHID, make one up.
         if "channelID" not in params:
-            return self._error(400, "Invalid arguments")
+            params["channelID"] = uuid.uuid4()
```

Comment:
Shouldn't this be a uuid.uuid4().hex so that its a uuid string rather than an object?